src/components/MobileHeader/MobileHeader.jsx

```diff
@@ -77,7 +77,7 @@ const MobileHeader = ({
     }
 
     document.addEventListener('mousedown', handleClick);
-    document.addEventListener('keyup', handleSearchEnter);
+    //document.addEventListener('keyup', handleSearchEnter);
```

Comment:
This solves the problem but I think we should just remove this line, also remove the 
`document.removeEventListener('keyup', handleSearchEnter);` on line 86
and just remove the handleSearchEnter entirely - as its no longer being used.